package.json

```
@@ -10,6 +10,8 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@apollo/client": "^3.12.5",
+    "graphql": "^16.10.0",
```

Usar a versão especifica da lib e reinstalar.
```suggestion
    "graphql": "16.10.0",
```

---

Okey, usarei só a libreria específica